documentation/cobigen-eclipse_debugging.asciidoc

```diff
@@ -33,7 +33,7 @@ image:images/debugging/osgi-console.png[]
 
 1. Right-click on the test you want to debug and select "Debug As" and then "Debug Configurations".
 
-2. In the "Debug Configurations" window, click on "New Configuration" to create a new test configuration.
+2. In the "Debug Configurations" window, select "JUni Plug-in Test" and click on "New Configuration" to create a new test configuration.
 
 3. Make sure that "Run in UI thread" is not selected.
```

Comment:
These steps can be removed with the latest changes by @maybeec 
Now we need to run/debug as SWT-Bot Test again. (Run in UI Thread is not needed anymore)

Reply:
Can be ignored, does not work yet.